docs/integration_resources/resources/usd.md

```diff
@@ -0,0 +1,29 @@
+# USD
+
+Shift has developed a specialized catalog and utility plugins for using the USD API in Shift. 
+
+* [Shift_USD](https://github.com/Inbibo/Shift_USD)
+
+## Installation
+
+To start working with the USD API in Shift, please follow the following steps:
+
+* Download the necessary dependencies (Python & USD).
+* Clone the [Shift_USD](https://github.com/Inbibo/Shift_USD) repository into the preferred folder.
+* Add the path to the Shift_USD directory to the `PYTHONPATH` environment variable.
+* Add the path to the Shift_USD directory to the `SHIFT_CATALOG_PATH` environment variable.
+* Add the path of the *plugin.json* file found in the Shift_USD directory to the `SHIFT_PLUGIN_PATH` environment variable.

```

Comment:
I was thinking that, ideally, inside our public repos we should have a folder named "catalogs" and another one "plugins". At the moment, for USD, we only have the plugins one.

Also, we have to remember to properly update the main branches of our public repositories so they have the lastest released changes.

Reply:
@Inbibo/shift if we go with this structure, we might have to change the documentation a bit to properly refer to where files are located.

Reply:
I like the idea of clearly separating Plugins and Catalogs.
However, I do not see how this would affect the documentation here.
At the end of the day, the `SHIFT_PLUGIN_PATH` has to point to the `plugins.json` file and the `SHIFT_CATALOG_PATH` has to point to the folder where the catalog files are present.
These two paths do not rely on having any particular folder structure.
We may want to suggest a folder structure, but it is not really mandatory to follow it.

Reply:
I think that it is a good idea to have the catalogs and plugins each in a separate folder. Maybe we should create an issue to create a Shift sample repo, that has a simple plugin and a catalog with one node that users can use as a starter template for creating their own catalog and/or plugin

Reply:
So we leave it as it is for the moment. I'm resolving the conversation then.